session1/excercises/generics.ts

```diff
@@ -0,0 +1,11 @@
+class mobile{
+    constructor(public modelName: string,public price: number) {        
+    }
+}
+
+function samsung<T extends mobile>(c: T) {
```

Comment:
1. This is logically not a function. Please correct the code logically.
2. Why are we using ES5 function syntax instead of ES6 arrow function?